crates/forge/tests/e2e/features.rs

```diff
@@ -0,0 +1,30 @@
+use super::common::runner::{setup_package, test_runner};
+use indoc::indoc;
+use shared::test_utils::output_assert::assert_stdout_contains;
+
+#[test]
+fn features() {
```

Comment:
Will work with new Scarb

Reply:
>Will work with new Scarb

Do you mean "Will fork with the next scarb release"? Which one?

Reply:
The next one, not sure what the version would be